lib/metadata_json_lint.rb

```diff
@@ -8,17 +8,21 @@
 require 'metadata-json-lint/schema'
 require 'metadata-json-lint/version_requirement'
 
+MIN_PUPPET_VER = '4.10.0'.freeze
```

Comment:
I think it's cleaner to move this inside `module MetadataJsonLint` so it's not in the global scope.